components/brave_rewards/resources/extension/brave_rewards/manifest.json

```diff
@@ -11,7 +11,8 @@
   "permissions": [
     "storage",
     "tabs",
-    "chrome://favicon/*"
+    "chrome://favicon/*",
+    "*://*/*"
```

Comment:
please open security review for it

Reply:
This is one area that I meant to come back to. I'm going to see if we can limit this down to increase the chances of a passing security review (and of course to increase security 😃 )

Reply:
This is changed to `https://www.twitch.tv/*`